proto/agent/agent.proto

```diff
@@ -98,3 +98,8 @@ message Data {
     bytes data = 3;
 }
 
+// AuthenticationTokens contains a definition of auth tokens to be passed in grpc calls
+message AuthenticationTokens  {
```

Comment:
Why do we need to define this as a proto message? Can we just send the token itself?

Reply:
It may be future proof to have it in it's own message IMHO

Reply:
IMHO since we are not building a grpc service around this message, making it a message only makes it less flexible, so it's the opposite of future proof. @Sh4d1 can you elaborate?

Reply:
@caesarxuchao yep sorry I had a public API in mind, I deleted the comment when I realized my mistake 😅

Reply:
At early stages of feature development, I passed 2 values instead of one, but found a way how to pass single token only, so yes, I will remove it, since it no more required